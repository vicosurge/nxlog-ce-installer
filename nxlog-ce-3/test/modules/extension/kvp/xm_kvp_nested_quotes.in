key1:"value " value" key2:""value""
key1:"value " val\:ue" key2:""value""
key1="value ", val\=ue", key2=""value""
app_name=HealthCheck, app_instance_id=575d6eb0716c4a5b8f785ddc21022d8d, event=CheckCompleted, check=DNS, Parameters="Query=example.com; Timeout=2000 ms", run_id=1, result=Success, processing_time=1094, text="OK, resolved "example.com" to 10.1.2.20, 10.1.2.14"
text="OK, resolved "example.com" to 10.1.2.20, 10.1.2.14", app_name=HealthCheck, app_instance_id=575d6eb0716c4a5b8f785ddc21022d8d, event=CheckCompleted, check=DNS, Parameters="Query=example.com; Timeout=2000 ms", run_id=1, result=Success, processing_time=1094
