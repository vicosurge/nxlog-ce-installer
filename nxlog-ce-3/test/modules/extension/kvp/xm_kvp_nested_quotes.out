{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value \" value","key2":"\"value\""}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value \" val\\:ue","key2":"\"value\""}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value \", val\\=ue","key2":"\"value\""}
{"SourceModuleName":"in","SourceModuleType":"im_file","app_name":"HealthCheck","app_instance_id":"575d6eb0716c4a5b8f785ddc21022d8d","event":"CheckCompleted","check":"DNS","Parameters":"Query=example.com; Timeout=2000 ms","run_id":1,"result":"Success","processing_time":1094,"text":"OK, resolved \"example.com\" to 10.1.2.20, 10.1.2.14"}
{"SourceModuleName":"in","SourceModuleType":"im_file","text":"OK, resolved \"example.com\" to 10.1.2.20, 10.1.2.14","app_name":"HealthCheck","app_instance_id":"575d6eb0716c4a5b8f785ddc21022d8d","event":"CheckCompleted","check":"DNS","Parameters":"Query=example.com; Timeout=2000 ms","run_id":1,"result":"Success","processing_time":1094}
